src/components/inputs/toggle-input/toggle-switch.js

```diff
@@ -1,61 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
 import Icons from './icons';
-import styles from './toggle-input.mod.css';
-
-const getStateClassNames = (baseClass, size, isDisabled, isChecked) =>
-  classnames(styles[`${baseClass}-${size}`], {
-    [styles.active]: isChecked,
-    [styles.disabled]: isDisabled,
-  });
+import {
+  getToggleSwitchStyles,
+  getToggleButtonStyles,
+  getToggleBarStyles,
+} from './toggle-input.styles';
 
 export const ToggleButton = props => (
-  <Icons.ButtonDefault
-    className={getStateClassNames(
-      'toggle-button',
-      props.size,
-      props.isDisabled,
-      props.isChecked
-    )}
-  />
-);
-
-export const ToggleBar = props => (
-  <Icons.BarDefault
-    className={getStateClassNames(
-      'toggle-bar',
-      props.size,
-      props.isDisabled,
-      props.isChecked
-    )}
-  />
+  <Icons.ButtonDefault css={getToggleButtonStyles(props)} />
 );
-
 ToggleButton.displayName = 'ToggleButton';
 ToggleButton.propTypes = {
-  size: PropTypes.string.isRequired,
+  size: PropTypes.oneOf(['small', 'big']).isRequired,
```

Comment:
unrelated: weird how we use 'small' and 'big' here and yet 's' and 'l' in LoadingSpinner.